src/main/scala/com/fortysevendeg/android/sqlite/resultset/SQLDroidResultSet.scala

```diff
@@ -9,6 +9,7 @@ import java.util.Calendar
 
 import android.database.Cursor
 import com.fortysevendeg.android.sqlite._
+import com.fortysevendeg.android.sqlite.data.{SQLDroidBlob, SQLDroidClob}
```

Comment:
No more changes in this file and new imports? Unused imports? It's a little weird :P